doc/scenario.md

```diff
@@ -666,6 +667,9 @@ Output facts:
 
 ### failover_promote
 
+Promotes leaders according to specified
+[`cartridge_failover_promote_params`](/doc/rolling_update.md#leaders-promotion).
```

Comment:
Let's use link to section with this variant of failover promote (Using `failover_promote` step)